packages/gatsby/src/utils/test-require-error.ts

```diff
@@ -16,6 +16,6 @@ export const testRequireError = (moduleName: string, err: any): boolean => {
       `\\$&`
     )}`
   )
-  const firstLine = err.toString().split(`\n`)[0]
+  const firstLine = err.toString().replace(/\\\\/g, `\\`).split(`\n`)[0]
   return regex.test(firstLine)
```

Comment:
```suggestion
  const [ firstLine ] = err.toString().split(`\n`)
  return regex.test(firstLine.replace(`\\\\`, `\\`))
```
Rather than running the regex over the whole error string, we can do a simple string replace on the single line that we care about.